cdap-ui/app/cdap/components/ResourceCenter/index.js

```diff
@@ -91,74 +126,37 @@ export default class ResourceCenter extends Component {
   };
   toggleWizard(wizardName) {
     this.setState({
-      [wizardName]: !this.state[wizardName],
+      activeWizard: wizardName,
       error: null
     });
   }
   closeWizard(wizardContainer) {
     ReactDOM.unmountComponentAtNode(wizardContainer);
   }
   getWizardToBeDisplayed() {
-    if (this.state.createStreamWizard) {
+    if (!this.state.activeWizard) {
+      return null;
+    } else if (this.state.activeWizard === 'createStreamWizard') {
       return (
         <StreamCreateWithUploadWizard
           isOpen={this.state.createStreamWizard}
```

Comment:
I think this needs to be changed to `isOpen={true}`, 'cause right now the Create Stream wizard doesn't open

Reply:
great catch! 👍